microservices/nris_api/backend/migrations/versions/43cd9db1c73a_.py

```diff
@@ -0,0 +1,75 @@
+    op.create_table_comment('document', 'A document is any type of additional documentation that has been generated during the INSPECTION process and is attached as part of the INSPECTION record. An example would be a photograph that was taken of the effluent being released during the inspection.')
+    op.create_table_comment(
+        'inspection', 'An inspection, otherwise known as a type of ASSESSMENT in NRIS, is an activity carried out by Mines Inspectors to ensure a Mine, and its relating mining activities, are in compliance with regulations in BC.')
+    op.create_table_comment(
+        'inspection_type', 'Categories of mine inspection types. This most directly related to the type of inspector and does not constrain potential resulting orders.')
+    op.create_table_comment('legislation_act_section',
+                            'Contains a list of sections (or provisions of the act); i.e. "1.9.1", "1.5.1", etc.')
+    op.create_table_comment('inspected_location', 'Contains the high-level details of an observation(s) found during an inspection. Note, one observation is either a ""Stop"" or ""General"", each observation can result in one or more ""types of observations"", i.e. documents being issued (order, warning, advisory, request).')
+    op.create_table_comment('inspection_document_xref',
+                            'Contains a reference between inspection documents and the details of the documents.')
+    op.create_table_comment('inspected_location_document_xref',
+                            'Contains a reference between inspected location documents and the details of the documents.')
+    op.create_table_comment('order_advisory_detail', 'For each inspection observation, this table contains details of advisories issued by an inspector. An advisory is a written notification to a person that draws attention to a specific regulatory requirement. An advisory is used where the proponent is in compliance at the moment of inspection but may be at risk of future non-compliance.')
+    op.create_table_comment('order_request_detail', 'For each inspection observation, this table contains requests from the inspector to the Chief Gold Commissioner to issue an order under the Mineral Tenure Act or the Coal Act. Also included are responses to the issued orders.')
+    op.create_table_comment('order_stop_detail', 'For each inspection observation, this table contains details of an order issued by an inspector. An order is a written, legal instrument issued by an inspector to address non-compliance with a Regulatory Requirement and/or to reduce and manage risk. This is the most common C&E tool that will be used by inspectors in all disciplines.')
+    op.create_table_comment('order_warning_detail', 'For each inspection observation, this table contains details of a warning issued by an inspector.  A warning is a written notification to a person that is not in compliance with a specific Reulatory Requirement at the time of inspection.')
```

Comment:
Typo in Reulatory (Regulatory)

Reply:
Fixed with the smallest commit I've ever made.